.github/workflows/deploy-beta.yml

```diff
@@ -12,31 +12,20 @@ jobs:
       CI: true
     steps:
       - name: Create GitHub deployment
-        uses: bobheadxi/deployments@v0.4.3
+        uses: bobheadxi/deployments@v0.6.2
         id: deployment
         with:
           step: start
           token: ${{ github.token }}
           env: beta
 
       - uses: actions/checkout@v2
-      - name: Use Node.js 14.x
-        uses: actions/setup-node@v1
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: 16.x
+          cache: npm
 
-      # From: https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows#example-using-the-cache-action
-      - name: Cache node modules
-        uses: actions/cache@v2
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: ~/.npm # npm cache files are stored in `~/.npm` on Linux/macOS
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
```

Comment:
The cache step is now an option in actions/node@v2 when using `with: cache: npm`

https://github.com/unicorn-utterances/unicorn-utterances/runs/4717609957#step:7:15